website/content/docs/ecs/manual/secure-configuration.mdx

```diff
@@ -0,0 +1,215 @@
+---
+layout: docs
+page_title: Secure Configuration - AWS ECS
+description: >-
+  Manual Secure Confguration of the Consul Service Mesh on AWS ECS (Elastic Container Service).
+---
+
+# Secure Configuration
+
+For a production-ready installation of Consul on ECS, you will need to make sure that the cluster is secured.
+A secure Consul cluster should include the following:
+
+1. [TLS Encryption](/docs/security/encryption#rpc-encryption-with-tls) for RPC communication between Consul clients and servers.
+1. [Gossip Encryption](/docs/security/encryption#gossip-encryption) for encrypting gossip traffic.
+1. [Access Control (ACLs)](/docs/security/acl) for authentication and authorization for Consul clients and services on the mesh.
+
+-> **NOTE:** This page assumes that you have already configured your Consul server with the above features.
```

Comment:
```suggestion
-> **NOTE:** In this topic, we assume that you have already configured your Consul server with the security-related features.
```
Nitpick, but I like referring to "topics" instead of "pages" just in case we need to copy and paste section and the context is no longer the whole page.